conda/core/solve.py

```diff
@@ -1341,8 +1341,9 @@ def __init__(
         self.r = None
 
         # Group 4. Mutable working containers
-        self.specs_map = {}
-        self.solution_precs = tuple(self.prefix_data.iter_records())
+        self.specs_map = odict()
```

Comment:
We've removed usage of `odict` in conda (see https://github.com/conda/conda/pull/12254)
```suggestion
        self.specs_map = {}
```

Reply:
(since ordinary Python dicts are ordered now)